content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,27 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Display labels instead of codes in your product exports
+::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/exports.html#update-an-export-profile"
+:::
+
+To save time and improve the reading of exported files, we added an option in our XLSX and CSV export functionalities to display attribute labels instead of codes in the exported files.
+Under your product export profiles and also in the quick export, you will see a new button called `With labels`. If enabled, this button will allow you to export labels for your product data. You can choose to display **readable column's headers** instead of attribute and property codes. In the file's content, you can now display the **attribute values** in a chosen locale instead of their codes.
```

Comment:
```suggestion
Under your product export profiles and also in the quick export, you will see a new button called `With labels`. If enabled, this button will allow you to export labels for your product data. You can choose to display **readable column's headers** instead of attribute and property codes. In the file's content, you can now display the **attribute values** in a chosen locale instead of their codes.  
  
![New Export Profile option](../img/Product-export-profile.png)  
```